app/models/catagory.rb

```diff
@@ -0,0 +1,6 @@
+class Catagory < ApplicationRecord
+  mount_uploader :icon, IconUploader
+
```

Comment:
Please validate user input in the `Item`, and `Catagory` models to make sure that anyone with bad intentions cannot compromise your app.